trio/_tests/verify_types.json

```diff
@@ -45,54 +45,32 @@
       }
     ],
     "otherSymbolCounts": {
-      "withAmbiguousType": 15,
-      "withKnownType": 231,
-      "withUnknownType": 236
+      "withAmbiguousType": 14,
+      "withKnownType": 244,
+      "withUnknownType": 224
     },
     "packageName": "trio",
     "symbols": [
-      "trio._core._exceptions.Cancelled",
-      "trio._core._exceptions.Cancelled.__str__",
-      "trio._util.NoPublicConstructor",
-      "trio._util.NoPublicConstructor.__call__",
-      "trio._util.Final.__new__",
       "trio.run",
-      "trio.open_nursery",
-      "trio._core._run.CancelScope",
-      "trio._core._run.CancelScope.cancelled_caught",
-      "trio._core._run.CancelScope.__exit__",
-      "trio._core._run.CancelScope.__repr__",
-      "trio._core._run.CancelScope.deadline",
-      "trio._core._run.CancelScope.cancel_called",
       "trio.current_effective_deadline",
       "trio._core._run._TaskStatusIgnored.__repr__",
       "trio._core._run._TaskStatusIgnored.started",
       "trio.current_time",
-      "trio._core._run.Nursery",
       "trio._core._run.Nursery.__init__",
       "trio._core._run.Nursery.child_tasks",
       "trio._core._run.Nursery.parent_task",
       "trio._core._run.Nursery.start_soon",
       "trio._core._run.Nursery.start",
       "trio._core._run.Nursery.__del__",
-      "trio.move_on_at",
-      "trio.move_on_after",
-      "trio.sleep_forever",
-      "trio.sleep_until",
-      "trio.sleep",
-      "trio.fail_after",
-      "trio._sync.Event",
       "trio._sync.Event.is_set",
       "trio._sync.Event.wait",
       "trio._sync.Event.statistics",
-      "trio._sync.CapacityLimiter",
```

Comment:
Most/all of these random classes being type complete is due to `Final.__new__` getting typed.